config-example/el/genesis-config.yaml

```diff
@@ -31,3 +31,4 @@ genesis_delay: ${GENESIS_DELAY}
 genesis_gaslimit: ${GENESIS_GASLIMIT}
 slot_duration_in_seconds: ${SLOT_DURATION_IN_SECONDS}
 electra_fork_epoch: ${ELECTRA_FORK_EPOCH}
+eof_activation_epoch: ${EOF_ACTIVATION_EPOCH}
```

Comment:
minor nit, add the `EOF_ACTIVATION_EPOCH` to the example env files pls and comment it out since its not standard yet